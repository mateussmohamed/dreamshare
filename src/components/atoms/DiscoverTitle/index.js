import styled from 'styled-components'

const DiscoverTitle = styled.h2`
  ${props => props.theme.typography.montserratStyle};
  font-size: 22px;
  font-weight: bold;
  text-align: center;
  color: ${props => props.theme.colors.greyishBrown};

  @media only screen and (min-width: 320px) {
    padding: 0 25px;
    margin: 0;
  }
`

export default DiscoverTitle
