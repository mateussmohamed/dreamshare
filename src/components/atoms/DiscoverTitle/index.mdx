---
name: DiscoverTitle
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import DiscoverTitle from './index'

# DiscoverTitle
<PropsTable of={DiscoverTitle} />

<Playground>
  <DiscoverTitle>
    SHARE YOUR HOLIDAY DREAM
  </DiscoverTitle>
</Playground>
