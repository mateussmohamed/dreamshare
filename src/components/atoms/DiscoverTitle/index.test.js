import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import DiscoverTitle from './index'

describe('[<DiscoverTitle />]', () => {
  const wrap = (props = {}) => shallow(<DiscoverTitle {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a Title' })

    expect(wrapper).toMatchSnapshot()
  })
})
