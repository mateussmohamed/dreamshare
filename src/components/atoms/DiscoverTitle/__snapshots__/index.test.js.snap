// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<DiscoverTitle />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "DiscoverTitle-sc-1yzz7od-0",
        "isStatic": false,
        "rules": Array [
          [Function],
          ";font-size:22px;font-weight:bold;text-align:center;color:",
          [Function],
          ";@media only screen and (min-width:320px){padding:0 25px;margin:0;}",
        ],
      },
      "displayName": "DiscoverTitle",
      "render": [Function],
      "styledComponentId": "DiscoverTitle-sc-1yzz7od-0",
      "target": "h2",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a Title
</StyledComponent>
`;
